simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
> the wrappers are supposed to be instantiated by Simulation object and this details are hidden from outside users.

Good point.  In addition, even without putting the arguments into CUDS, we would still have the issue with the docstring of `Simulation.__init__`:  A user new to the framework will need to use `get_wrapper(wrapper_name)` in order to fetch the class and its docstring.

Alternatively, this may be explicitly as for what the user should do:
```
cuds = CUDS()
sim = Simulation(cuds, LammpsWrapper(use_internal_interface=True))
```
Note: my understanding is that we should restrict a new engine wrapper instance being created for each individual `Simulation` instance in case it shares the same cuds with another `Simulation`, which could be a problem depending on how the engine wrapper is implemented (e.g. a file-io wrapper maybe initialised to look at a particularly file directory and different Simulation sharing the same wrapper ends on overwriting the files).